stl/debugger/STL.natvis

```diff
@@ -247,6 +247,45 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
   </Type>
 
 
+  <Type Name="std::unexpected&lt;*&gt;">
+      <Intrinsic Name="unex" Expression="_Unexpected"/>

```

Comment:
I had a conversation with the author of P2549 and we aggreed that we should also adapt the name of the exposition variable `E val_` to `E unex_`

This will be part of the next revision von P2549 so that I already implemented the updated visualizer.

It would have been confusing why we have paylod `val` and error `unex` once and error `val`  elsewhere